content/docs/gke/customizing-gke.md

```diff
@@ -7,56 +7,104 @@ weight = 2
 This guide describes how to customize your deployment of Kubeflow on Google 
 Kubernetes Engine (GKE) in Google Cloud Platform (GCP).
 
+## Customizing Kubeflow before deployment
+
+The Kubeflow deployment process is divided into two steps, **build** and 
+**apply**, so that you can modify your configuration before deploying your 
+Kubeflow cluster.
+
+Follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/).
+When you reach the 
+[setup and deploy step](/docs/gke/deploy/deploy-cli/#set-up-and-deploy),
+**skip the `kfctl apply` command** and run the **`kfctl build`** command 
+instead, as  described in that step. Now you can edit the configuration files 
+before deploying Kubeflow.
+
+## Customizing an existing deployment
+
+You can also customize an existing Kubeflow deployment. In that case, this 
+guide assumes that you have already followed the guide to 
+[deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/) and have deployed
+Kubeflow to a GKE cluster.
+
 ## Before you start
 
-This guide assumes you have already set up Kubeflow with GKE. If you haven't done
-so, follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/).
+This guide assumes the following settings: 
 
-## Customizing Kubeflow
+* The `${KFAPP}` environment variable contains the name
+  of your Kubeflow application directory, as described in the guide to
+  [deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/):
 
-You can use [kustomize](https://kustomize.io/) to customize Kubeflow.
+  ```
+  export KFAPP=<your choice of application directory name>
```

Comment:
nit: I would say name of Kubeflow deployment; The directory is one "resource" that gets named after the deployment.

Reply:
Done.